### PR TITLE
Adds pagination to Github request.

### LINKED ISSUE #3: Pagination is not implemented
Unfortunately most of the Github APIs returning collections returns maximum 30 responses. This is documented here https://developer.github.com/v3/#pagination but unfortunately it is not mentioned in the particular APIs that are affected. So when reading https://developer.github.com/v3/pulls/#list-pull-requests  it's not at all obvious that pagination is in operation and that you should add another loop through pages. 

Periodic labeler does not have pagination implemented, therefore the result is that in big repositories with > 30 PRs only the OLDEST prs will be checked (that's what is the sorting order). If all those 30 PRs are properly labelled, then the plugin will stop working and will not pick any new PRs.  There is an interesting behaviour of that - if there are unlabelled old PRs, they will get updated, so - in fact - they will disappear from the "oldest" list and possibly some new PRs that will be checked in the new pass. However once you have 30 oldest, properly labelled PRs, the plugin stops working. That's really annoying and difficult to detect problem - especially if your repo does not have a lot of PRs. 

Pagination is easy. It is described here: https://github.com/google/go-github/#pagination

Pull request with a fix follows shortly.

### PR DESCRIPTION
**Pull request by potiuk**
Fixes #3.

Unfortunately most of the Github APIs returning collections returns maximum 30
responses. This is documented here https://developer.github.com/v3/#pagination
but unfortunately it is not mentioned in the particular APIs that are affected.
So when reading https://developer.github.com/v3/pulls/#list-pull-requests it's
not at all obvious that pagination is in operation and that you should add
another loop through pages.

Periodic labeler does not have pagination implemented, therefore the result is
that in big repositories with > 30 PRs only the OLDEST prs will be checked
(that's what is the sorting order). If all those 30 PRs are properly labelled,
then the plugin will stop working and will not pick any new PRs. There is an
interesting behaviour of that - if there are unlabelled old PRs, they will get
updated, so - in fact - they will disappear from the "oldest" list and possibly
some new PRs that will be checked in the new pass. However once you have 30
oldest, properly labelled PRs, the plugin stops working. That's really annoying
and difficult to detect problem - especially if your repo does not have a lot
of PRs.

Pagination is easy. It is described here:
https://github.com/google/go-github/#pagination

### CODE REVIEW

**Comment by kaxil** [comment]
@paulfantom Would love to get this merged. We want to use it for https://github.com/apache/airflow project

File: main.go [hunk 1]
```diff
@@ -9,18 +9,18 @@ import (
 	"github.com/golang/glog"
 	"golang.org/x/oauth2"
 
-	"github.com/google/go-github/v28/github"
+	"github.com/google/go-github/github"
```
Comment (potiuk) [inline main.go]: The v28 did not work for me when I tested it locally. I've read in the docs of go-github that you can use either v28 or github, not 100% sure what is the difference. Happy to come back to v28 if that's a problem.

File: main.go [hunk 2]
```diff
@@ -9,18 +9,18 @@ import (
...
 	"gopkg.in/yaml.v2"
 )
 
 func contains(slice []string, item string) bool {
-    set := make(map[string]struct{}, len(slice))
-    for _, s := range slice {
-        set[s] = struct{}{}
-    }
+	set := make(map[string]struct{}, len(slice))
```
Comment (potiuk) [inline main.go:17]: I run gofmt with the project and it formatted it this way. I hope it's OK for you in one PR.

File: main.go (hunk 1 above)
Comment (potiuk) [reply main.go]: OK. I looked at the gomod and I understand now. Reverted.

**Comment by paulfantom** [comment]
Awesome! Thanks!

**Pull request merged**